org.mixedrealitytoolkit.accessibility/CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [1.0.2-development] - 2024-03-20
+## [1.0.3-development] - 2024-04-17
+
+### Fixed
+
+* Reduced package description to support for UPM package publishing in the Unity Asset Store.
```

Comment:
Correct, from a merge from main because in a maintainers meeting someone requested to keep the branch up to date with main but I'll do merges from main in separate PRs from now on.  Resolving.

Reply:
Yes, keep the branch up-to-date...but do this with a separate PR, do not mixed with your changes.  Otherwise, it is difficult to differentiate between what you changed and what others changed.  

So next time keep this branch up-to-date with standalone merge PR, and do not add your own changes

Reply:
Will do.